RobotCode/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,7 +33,9 @@
 
 public class RobotContainer
 {
-
+	public static Joystick primaryController;
```

Comment:
Make this private. Variables are always private. Also make it not static. There is no reason for these variables to be static. Also you can condense both joystick declarations into `private Joystick primaryController, secondaryController`.